src/app/routes/utils/regex/utils/index.js

```diff
@@ -101,9 +101,7 @@ export const getHomePageRegex = services => {
   if (isLive()) {
     homePages = services.filter(service => homePageServices.includes(service));
   } else {
-    homePages = services.filter(
-      service => !servicesWithVariants.includes(service),
-    );
+    homePages = services;
```

Comment:
This entire else statement can be removed, since homePages are already set to services on line 100